stl/set_environment.bat.in

```diff
@@ -0,0 +1,3 @@
+@set PATH="%~dp0out\bin\@VCLIBS_I386_OR_AMD64@;%PATH%"

```

Comment:
Can you add copyright/license banners to these files? For `bat`, `::` works for comments.